src/gui/gui.pri

```diff
@@ -12,6 +12,7 @@ HEADERS += \
     $$PWD/cookiesdialog.h \
     $$PWD/cookiesmodel.h \
     $$PWD/deletionconfirmationdialog.h \
+    $$PWD/doublespinbox.h \
```

Comment:
>Please check and advise why appveyor build is failing, as it builds fine for me on Linux & Windows.

You'll need to add `doublespinbox.cpp` to the other list in this file.

Reply:
Thank you @Chocobo1, it helped.